app/factories/bulkrax/object_factory.rb

```diff
@@ -163,22 +163,22 @@ def create_file_set(attrs)
         uploaded_file = ::Hyrax::UploadedFile.find(uploaded_file_id)
         next if uploaded_file.file_set_uri.present?
 
-        create_file_set_actor(attrs, work, work_permissions, uploaded_file)
+        create_file_set_actor(attrs, work, work_permissions, uploaded_file, type: 'uploaded')
       end
       attrs['remote_files']&.each do |remote_file|
-        create_file_set_actor(attrs, work, work_permissions, nil, remote_file)
+        create_file_set_actor(attrs, work, work_permissions, remote_file, type: 'remote')
```

Comment:
passing whatever file type we're looping over and handling it as `file` in `#create_file_set_actor` below. also passing ta `type` parameter that we can  check against instead of setting "uploaded_file" or "remote_file" to nil.

this also sets us up for the possible future case that we also handle the `file_url` header on a FileSet row.